src/components/notifications/Whapi.vue

```diff
@@ -0,0 +1,23 @@
+<template>
+    <div class="mb-3">
+        <label for="whapi-api-url" class="form-label">{{ $t("API URL") }}</label>
+        <input id="whapi-api-url" v-model="$parent.notification.whapiApiUrl" type="text" class="form-control" required>
+    </div>
+
+    <div class="mb-3">
+        <label for="whapi-auth-token" class="form-label">{{ $t("Token") }}</label>
+        <input id="whapi-auth-token" v-model="$parent.notification.whapiAuthToken" type="text" class="form-control" required>
```

Comment:
I have never worked with said provider.
Do you think a helptext is nessesary on how to get this "Token"?